api/contract.go

```diff
@@ -44,6 +45,23 @@ type createFilterResponse struct {
 	model.Filter
 }
 
+// createFilterRequest is the request body for POST /filters
+type createFilterOutputsRequest struct {
+	State     string             `bson:"state" json:"state"`
+	Downloads model.FilterOutput `bson:"downloads"      json:"downloads"`
+}
+
+func (r *createFilterOutputsRequest) Valid() error {
+
+	return nil
+}
```

Comment:
Do we need to introduce some logic to validate the `POST` request?

Reply:
A very basic validation has been addressed in validateFilterOutput